common/actions/config.js

```diff
@@ -1,41 +1,45 @@
 // @flow
+import {
+  NODE_CHANGE,
+  LANGUAGE_CHANGE,
+  GAS_PRICE_CHANGE
+} from './configConstants';
 
 export type ChangeNodeAction = {
-  type: 'CONFIG_NODE_CHANGE',
+  type: NODE_CHANGE,
```

Comment:
Problem here is type of `type` is not string constant but string (since you not using inline value). This approach is definitely working with differently shaped actions, i.e. https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/actions/ens.js